### ISSUE TITLE
Parsing error in get_artist_albums

### ISSUE THREAD

**Issue by fabi321** [sigma67/ytmusicapi#271]
On some channels this happens occasionally, but I finally fond a channel where it happens every single time. The artist is `Eluveitie`, channel id `UClV0x3k5Oxm7vREoLEOPeGg`.
After using `get_artist` to get general info, it returns an album list containing a `params` string. When running `get_artist_albums`, it fails with the following stacktrace:
```
  File "/main.py", line 79, in get_albums_for_artist
    return ytmusic.get_artist_albums(artist['channelId'], artist['albums']['params'])
  File "/.venv/lib/python3.10/site-packages/ytmusicapi/mixins/browsing.py", line 446, in get_artist_albums
    results = nav(response, SINGLE_COLUMN_TAB + SECTION_LIST_ITEM + GRID_ITEMS)
  File "/.venv/lib/python3.10/site-packages/ytmusicapi/parsers/utils.py", line 155, in nav
    raise err
  File "/.venv/lib/python3.10/site-packages/ytmusicapi/parsers/utils.py", line 149, in nav
    root = root[k]
KeyError: 'content'
```
looking into the response of https://github.com/sigma67/ytmusicapi/blob/master/ytmusicapi/mixins/browsing.py#L254, I got the following content:
```json
{
  "responseContext": {...},
  "contents": {
    "singleColumnBrowseResultsRenderer": {
      "tabs": [
        {
          "tabRenderer": {
            "trackingParams": "..."
          }
        }
      ]
    }
  },
  "header": {
    "musicHeaderRenderer": {
      "title": {
        "runs": [
          {
            "text": "Eluveitie"
          }
        ]
      },
      "trackingParams": "..."
    }
  },
  "trackingParams": "..."
}
```
Version: `0.21.0`
There are other channels, where the same issue happened sometimes, but not always.

**Comment by sigma67** [comment]
I believe this is related to a recently fixed issue where unauthenticated users would sometimes be served YouTube's consent page instead of the actual page content. I discovered this issue last week when I observed tests failing (seemingly) randomly. After spending some time debugging I found that it only happens for unauthenticated users and fixed the root cause in 7c242ba. 

Please pull and install the latest master for now, a new release containing the fix will be pushed this week.

**Comment by fabi321** [comment]
I just checked this and can confirm that the issue is fixed on the latest master version
